awx/ui_next/src/components/FormField/sortErrorMessages.js

```diff
@@ -0,0 +1,35 @@
+export default function sortErrorMessages(error, formValues = {}) {
+  if (!error) {
+    return {};
+  }
+
+  const fieldErrors = {};
+  let formErrors = [];
+  if (
+    error?.response?.data &&
+    typeof error.response.data === 'object' &&
+    Object.keys(error.response.data).length > 0
+  ) {
+    Object.keys(error.response.data).forEach(fieldName => {
+      const errors = error.response.data[fieldName];
+      if (!errors) {
+        return;
+      }
+      const errorsArray = Array.isArray(errors) ? errors : [errors];
+      if (typeof formValues[fieldName] === 'undefined') {
+        formErrors = [...formErrors, ...errorsArray];
+      } else {
+        fieldErrors[fieldName] = errorsArray.join('; ');
```

Comment:
Will this display the error message with a `;` at the end of it under the form field?

Reply:
If there are multiple errors, it will put `; ` between each of them